api/http/handler/endpoints/endpoint_agent_browse_docs.go

```diff
@@ -0,0 +1,25 @@
+package endpoints
+
+/// This feature is implemented in the agent API and not directly here.
+/// However, it's proxied.  So we document it here.
+
+// @summary Upload a file under a specific path on the file system of an environment (endpoint)
```

Comment:
This seems to go into the root portainer docs folder, should it go in a particular one? Maybe docker/{environmentId} ?

Reply:
This is the doc copied from our manual patching process.